cmd/run.go

```diff
@@ -414,6 +388,36 @@ a commandline interface for interacting with it.`,
 	},
 }
 
+func reportUsage(execScheduler *local.ExecutionScheduler) error {
+	execPlan := execScheduler.GetExecutionPlan()
+	executorConfigs := execScheduler.GetExecutorConfigs()
+	execState := execScheduler.GetState()
+
+	executors := make(map[string]int)
+	for _, ec := range executorConfigs {
+		executors[ec.GetType()]++
+	}
+
+	body, err := json.Marshal(map[string]interface{}{
+		"k6_version": consts.Version,
```

Comment:
Should we use `consts.FullVersion()` here instead and remove the `goos`/`goarch` properties since they're in the full version? It might be handy to keep them apart, but the full version also includes the build info, which might be useful.

Reply:
I am for using `consts.FullVersion()` and keeping `goos` and `goarch` separate

Reply:
It's a bit of duplication, but sure, I changed it in b0b153a2.

Reply:
Actually, we should probably avoid using composite strings here. Otherwise aggregating the results would become a parsing nightmare...

So I'll revert this to `const.Version` and if we need additional info let's add it in separate fields.

Reply:
Yes, also we don't want to expose any potentially sensitive things like commit IDs/tag names from the full version information. Or break backwards compatibility (the same backend will receive pings from all sorts of k6 versions). :+1: for `consts.Version`